package.json

```diff
@@ -10,15 +10,16 @@
   "version": "7.1.0",
   "author": "Beau Gunderson <beau@beaugunderson.com> (https://beaugunderson.com/)",
   "license": "MIT",
-  "main": "dist/ip-address.js",
-  "types": "dist/ip-address.d.ts",
+  "main": "dist/cjs/ip-address.js",
+  "module": "dist/esm/ip-address.js",
   "scripts": {
     "docs": "documentation build --github --output docs --format html ./ip-address.js",
-    "prepublishOnly": "tsc",
+    "build": "tsc -p tsconfig.json && tsc -p tsconfig-cjs.json",
+    "prepublishOnly": "npm run build",
     "release": "release-it",
-    "test-ci": "nyc mocha",
-    "test": "mocha",
-    "watch": "mocha --watch"
+    "test-ci": "nyc jest --config jest.config.js --ci",
```

Comment:
would prefer to continue using `mocha`; what issues did you have with it? i'm happy to fix that aspect

Reply:
Ran into mocha not liking the module config in package.json and tried for about 20 minutes then gave up. Happy for it to be fixed however and really I shouldn't have put it into the PR.

Reply:
I've reverted that commit with 742410e so you can see how it was working beforehand.